Java/Java.sublime-syntax

```diff
@@ -8,19 +8,42 @@ file_extensions:
 scope: source.java
 contexts:
   main:
-    - match: '^\s*(package)\b(?:\s*([^ ;$]+)\s*(;)?)?'
-      scope: meta.package.java
-      captures:
-        1: keyword.other.package.java
-        2: storage.modifier.package.java
-        3: punctuation.terminator.java
-    - match: '^\s*(import)\b(?:\s*([^ ;$]+)\s*(;)?)?'
-      scope: meta.import.java
-      captures:
-        1: keyword.other.import.java
-        2: storage.modifier.import.java
-        3: punctuation.terminator.java
+    - include: package
+    - include: import
     - include: code
+  package:
+    - match: '\bpackage\b'
+      scope: keyword.control.package.java
+      push:
+        - meta_scope: meta.package.java
+        - match: '[\w\.]+'
+          scope: keyword.other.package.java
+        - match: ;
+          scope: punctuation.terminator.java
+          pop: true
+        - match: (?=\n)
+          pop: true
+  import:
+    - match: \bimport\b
+      scope: keyword.control.import.java
+      push:
+        - meta_scope: meta.import.java
+        - match: \bstatic\b
+          scope: storage.modifier.static.java
+          push:
+            - match: '[\w\.]+'
+              scope: keyword.other.static.import.java
```

Comment:
In Java imports are always fully qualified class paths. e.g., `foo.bar.SomeClass`. So maybe `support.class.import` makes sense. You can either add the class as an import and reference it as `SomeClass` in the file, or skip adding the import and use the fully qualified path.

Given the above, maybe the package path be `support.namespace`?

Static imports are always fully qualified static methods. e.g., `foo.bar.SomeClass.myStaticMethod`. They are not actual invocations so not sure if `variable.function` makes sense. How about `support.function.import`?